torch/_inductor/codecache.py

```diff
@@ -2184,12 +2184,23 @@ class CppPythonBindingsCodeCache(CppCodeCache):
             static_assert(std::is_pointer<T>::value, "arg type must be pointer or long");
             return static_cast<T>(_torchinductor_pyobject_tensor_data_ptr(PyTuple_GET_ITEM(args, n)));
         }
+
+        #ifdef _WIN32
+        template <> inline int64_t parse_arg<int64_t>(PyObject* args, size_t n) {
+            auto result = PyLong_AsSsize_t(PyTuple_GET_ITEM(args, n));
+            if(unlikely(result == -1 && PyErr_Occurred()))
+                throw std::runtime_error("expected int arg");
+            return result;
+        }
+        #else
         template <> inline long parse_arg<long>(PyObject* args, size_t n) {
```

Comment:
```suggestion
        template <> inline int64_t parse_arg<int64_t>(PyObject* args, size_t n) {
```

If you make this change, can you just remove the ifdef entirely?  This should be a no-op change on linux.